src/com/intellij/idea/plugin/hybris/project/descriptors/DefaultHybrisProjectDescriptor.java

```diff
@@ -764,6 +777,45 @@ private void scanSubdirectories(
         }
     }
 
+    private void scanSubdirectoriesWSL(@NotNull final Map<DIRECTORY_TYPE, Set<File>> moduleRootMap, final Set<File> excludedFromScanning, final boolean acceptOnlyHybrisModules, @NotNull final Path rootProjectDirectory, @Nullable final TaskProgressProcessor<File> progressListenerProcessor) throws InterruptedException, IOException {
```

Comment:
method style can be slightly simplified:
```
private void scanSubdirectoriesWSL(@NotNull final Map<DIRECTORY_TYPE, Set<File>> moduleRootMap, final Set<File> excludedFromScanning, final boolean acceptOnlyHybrisModules, @NotNull final Path rootProjectDirectory, @Nullable final TaskProgressProcessor<File> progressListenerProcessor) throws InterruptedException, IOException {
        if (!Files.isDirectory(rootProjectDirectory)) return;

        try (final Stream<Path> stream = Files.list(rootProjectDirectory)) {
            final var moduleRoots = stream
                .filter(Objects::nonNull)
                .filter(Files::isDirectory)
                .filter(Predicate.not(this::isDirectoryExcluded))
                .filter(file -> !Files.isSymbolicLink(file) || followSymlink)
                .map(Path::toFile)
                .toList();
            for (final var moduleRoot : moduleRoots) {
                findModuleRoots(moduleRootMap, excludedFromScanning, acceptOnlyHybrisModules, moduleRoot, progressListenerProcessor);
            }
        }
    }
```

Reply:
Done :)